package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "js-yaml",
-  "version": "3.12.0",
+  "version": "3.13.0",
```

Comment:
Not sure if I'm supposed to version bump or not...

Reply:
Don't change configs.